source/security.txt

```diff
@@ -131,51 +131,68 @@ Environment Variables
 ~~~~~~~~~~~~~~~~~~~~~
 
 The following code shows how to create a connection URI to
-authenticate with ``MONGODB-AWS`` when obtaining credentials from environment variables:
+authenticate with ``MONGODB-AWS`` when obtaining credentials from environment
+variables. Ensure you have your environment variables specified before running
+this code.
 
 .. literalinclude:: /includes/authentication.cpp
    :language: cpp
    :copyable: true
    :start-after: // start-aws-environment
    :end-before: // end-aws-environment
 
+To learn more about authenticating with AWS by using environment
+variables, see :ref:`cpp-mongo-aws-environment`.
```

Comment:
As part of removing the "Environment Variables" section, recommend moving this line to immediately after "To learn more about authenticating with AWS, see :ref:`cpp-mongo-aws` in the Authentication guide." on line 116.

Reply:
Addressed by https://github.com/mongodb/docs-cpp/pull/25/files#r1683312780.